makefile.gen

```diff
@@ -141,7 +141,6 @@ out/symbol.txt: out/rom.out
 
 out/rom.out: out/sega.o out/cmd_ $(LIBMD)
 	$(CC) -m68000 -B$(BIN) -n -T $(GDK)/md.ld -nostdlib out/sega.o @out/cmd_ $(LIBMD) $(LIBGCC) -o out/rom.out -Wl,--gc-sections -flto
-	$(RM) out/cmd_

```

Comment:
does the out/cmd_ file get automatically deleted after compilation ?

Reply:
It was being deleted after compilation. It is also a dependancy for the user application. So, if re-run make -f ../makefile.gen it will link the user application again even if there are no changes because this is how out/cmd_ gets created. Maybe a better way is just to pass the object file names directly to the linker command line as we know what they are?